ShowMySky/CMakeLists.txt

```diff
@@ -4,12 +4,14 @@ endif()
 
 add_definitions(-D_USE_MATH_DEFINES)
 add_library(ShowMySky SHARED
-             api/AtmosphereRenderer.cpp
+             ShowMySky/AtmosphereRenderer.cpp
```

Comment:
`api` subdirectory should remain, otherwise it's a mess to understand the directory structure. I suppose you're trying to keep the `#include` directives in the user code intact by this change. In this case this can be achieved by changing `api` → `api/ShowMySky` instead of `api` → `ShowMySky` (and a corresponding tweak of the `...INCLUDE_DIRECTORIES`).

Reply:
Done